scheduler-workflow/workflow.go

```diff
@@ -58,7 +54,13 @@ func main() {
 	}
 	log.Println("Workflow worker started")
 
-	go startLonghaulWorkflow(ctx, client)
+	// make sure to clean up any old workflows from previous runs
+	for i := 0; i < 100; i++ {
+		wfClient.TerminateWorkflow(ctx, fmt.Sprintf("longhaul-instance-%d", i))
+		wfClient.PurgeWorkflow(ctx, fmt.Sprintf("longhaul-instance-%d", i))
+	}
```

Comment:
I think I'd like to see safer handling of workflows rather than mass firing off terminate/purge requests, possibly attempting to retrieve those 100 workflow IDs and if found provide some debug logging